src/Playroom/SettingsPanel/SettingsPanel.tsx

```diff
@@ -73,101 +74,144 @@ const KeyboardShortcut = ({
+
   return (
-    <ToolbarPanel data-testid="frame-panel">
-      <Stack space="large" dividers>
-        <fieldset className={styles.fieldset}>
-          <legend>
-            <Heading level="3">Editor Position</Heading>
-          </legend>
-          <div className={styles.radioContainer}>
-            {['Bottom', 'Right'].map((option) => (
-              <div key={option}>
-                <input
-                  type="radio"
-                  name="editorPosition"
-                  id={`editorPosition${option}`}
-                  value={option.toLowerCase()}
-                  title={option}
-                  checked={option.toLowerCase() === editorPosition}
-                  onChange={() =>
-                    dispatch({
-                      type: 'updateEditorPosition',
-                      payload: {
-                        position: option.toLowerCase() as EditorPosition,
-                      },
-                    })
-                  }
-                  className={styles.realRadio}
-                />
-                <label
-                  htmlFor={`editorPosition${option}`}
-                  className={styles.label}
-                  title={option}
-                >
-                  <span className={styles.labelText}>
-                    {positionIcon[option.toLowerCase() as EditorPosition]}
-                  </span>
-                </label>
-              </div>
-            ))}
-          </div>
-        </fieldset>
-
-        <fieldset className={styles.fieldset}>
-          <legend>
-            <Heading level="3">Color Scheme</Heading>
-          </legend>
-          <div className={styles.radioContainer}>
-            {['Light', 'Dark', 'System'].map((option) => (
-              <div key={option}>
-                <input
-                  type="radio"
-                  name="colorScheme"
-                  id={`colorScheme${option}`}
-                  value={option.toLowerCase()}
-                  title={option}
-                  checked={option.toLowerCase() === colorScheme}
-                  onChange={() =>
-                    dispatch({
-                      type: 'updateColorScheme',
-                      payload: {
-                        colorScheme: option.toLowerCase() as ColorScheme,
-                      },
-                    })
-                  }
-                  className={styles.realRadio}
-                />
-                <label
-                  htmlFor={`colorScheme${option}`}
-                  className={styles.label}
-                  title={option}
-                >
-                  <span className={styles.labelText}>
-                    {colorModeIcon[option.toLowerCase() as ColorScheme]}
-                  </span>
-                </label>
-              </div>
+    <>
+      {localTitle === undefined ? null : (
+        <Helmet>
+          <title>{displayedTitle}</title>
+        </Helmet>
+      )}
+      <ToolbarPanel data-testid="frame-panel">
```

Comment:
Minor copy and paste issue (not from your PR)
```suggestion
      <ToolbarPanel data-testid="settings-panel">
```

Reply:
Ahh, nice catch!